modules/tensor_mechanics/src/base/TensorMechanicsApp.C

```diff
@@ -230,6 +231,7 @@ TensorMechanicsApp::associateSyntax(Syntax & syntax, ActionFactory & action_fact
   syntax.registerActionSyntax("DynamicTensorMechanicsAction", "Kernels/DynamicTensorMechanics");
   syntax.registerActionSyntax("PoroMechanicsAction", "Kernels/PoroMechanics");
   syntax.registerActionSyntax("TensorMechanicsAxisymmetricRZAction", "Kernels/AxisymmetricRZ");
+  syntax.registerActionSyntax("TensorMechanicsRSphericalAction", "Kernels/RSpherical");
```

Comment:
`AxisymmetricRZ` and `RSpherical` as block names directly under `Kernels` could be a bit more descriptive...